tutorials/hello/hello_client/config/default/config.json

```diff
@@ -1,11 +1,13 @@
 {
 	"system": {
-		"dataPath": "/Users/mona/.lisk/beta-sdk-app",
+		"dataPath": "/Users/iamtalha/.lisk/beta-sdk-app",
 		"keepEventsForHeights": 300,
 		"logLevel": "info"
 	},
 	"rpc": {
-		"modes": ["ipc"],
+		"modes": [
+			"ipc"
+		],
```

Comment:
`allowedMethods` is missing here, without it the app wont start:

```
2023-08-22T09:22:39.506Z INFO Monas-MacBook-Pro.local application 52364 helloInfo Loading in-memory plugin
2023-08-22T09:22:39.514Z INFO Monas-MacBook-Pro.local engine 52364 [status=error err=Requested method or namespace is disabled in node config.] Failed to handle IPC request
    Error: Requested method or namespace is disabled in node config.
2023-08-22T09:22:39.522Z INFO Monas-MacBook-Pro.local application 52364 [errorCode=2 message=process.exit] Application shutdown started
```

Reply:
Please use `./bin/run start --config=config/custom_config.json` to start the app as suggested in the [README](https://github.com/LiskHQ/lisk-sdk-examples/tree/development/tutorials/hello/hello_client#start-a-node)